tests/_phlop/app/pfm/test_pfm.py

```
@@ -0,0 +1,27 @@
+from phlop.proc import run
+from phlop.string import decode_bytes
+
+if __name__ == "__main__":
+    from phlop.app.pfm.check_events import get_evt_perf_code
+    from phlop.app.pfm.showevtinfo import get_evt_info
+
+    code = ""
+    key0, key1 = "[MULT_FLOPS]", "[ADD_SUB_FLOPS]"
+    for info in get_evt_info():
+        if key0 in info.umask:
+            for key, umask in info.umask.items():
```

Ruff

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)